static/src/javascripts/bootstraps/common.js

```diff
@@ -241,6 +243,24 @@ define([
                 ['resize', 'scroll', 'orientationchange'].forEach(function (event) {
                     bean.on(window, event, mediator.emit.bind(mediator, 'window:' + event));
                 });
+                (function () {
+                    var running = false;
+                    var lastScrollY = window.pageYOffset;
+                    bean.on(window, 'scroll', function () {
+                        if (!running) {
+                            running = true;
+                            // fastdom is a handy replacement for simply throttling with
+                            // `requestAnimationFrame` (since we're calling `window.pageYOffset`)
+                            fastdom.read(function () {
```

Comment:
I wouldn't use `fastdom` in this case but directly `requestAnimationFrame`. `fastdom.read` does batching of multiple calls and computes whether you need another frame or not. For a high performance optimization it's best to skip any unnecessary code and call primitives directly.